strimzi/README.md

```diff
@@ -0,0 +1,152 @@
+# Binding Event-driven Applications to an In-cluster Operator Managed Kafka Cluster
+
+## Status
+
+This sample is currently **EXPERIMENTAL**, which means that not all pieces are available in the community today.
+
+
+## Introduction
+
+This scenario illustrates binding two event-driven applications to an in-cluster Operator Managed Kafka cluster using the Service Binding Specification. 
+
+## Actions to Perform by Users in 2 Roles
+
+In this example there are 2 roles:
+
+* Cluster Admin - Installs the operators to the cluster
+* Application Developer - Creates a Kafka cluster, creates required Kafka resources, imports applications and creates a request to bind the applications to the Kafka cluster and the Kafka resources.
+
+### Cluster Admin
+
+The cluster admin needs to install 3 operators into the cluster:
+
+* Service Binding Operator
+* Strimzi Operator
+* Runtime Component Operator
+
+#### Install the Service Binding Operator
+
+Navigate to the **Operator** → **OperatorHub** in the OpenShift console. From the `Developer Tools` category, select the `Service Binding Operator` operator and install a `alpha` version. This would install the `ServiceBindingRequest` *custom resource definition* (CRD) in the cluster.
```

Comment:
Would be nice to include instructions that work outside of OpenShift as well

Reply:
yup, I agree.  I will open up an issue to follow up on this sample, and have instructions for vanilla k8s too.

Reply:
ref #2